CHANGELOG.md

```diff
@@ -7,6 +7,12 @@
 - Update `poi` layer based on ([`openpoiservice`]) [#246])
 - Remove `ideal_vgi_poi` layer in favor of new `poi` layer ([#246])
 
+
+### New Features
+
+- Add new parameter `layer` containing `name`, `description` and `data` fields to the API endpoint `indicator`. This enables to compute indicators for given data. ([#245])
```

Comment:
Should we make it clear here that this is only available via POST?

Reply:
.